### PR TITLE
delete dataset

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[RESPECT] Man kann als Admin öffentliche Datasets in Python wieder löschen](https://trello.com/c/WprlBxBJ/1167-respect-man-kann-als-admin-%C3%B6ffentliche-datasets-in-python-wieder-l%C3%B6schen)

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: examples/add_public_raster_dataset.ipynb [hunk 1]
```diff
@@ -309,12 +309,51 @@
...
+    "try:\n",
+    "    data = workflow.get_xarray(\n",
+    "        ge.QueryRectangle(\n",
+    "            [-180.0, -90.0, 180.0, 90.0],\n",
+    "            [time, time],\n",
+    "            resolution=[360. / 16, 180. / 16],\n",
+    "            )\n",
+    "    )\n",
+    "except Exception as e:\n",
+    "    print(e)"
```
[the remark this answers is not in the document]
Comment (michaelmattig) [reply examples/add_public_raster_dataset.ipynb:355]: for me it prints:

> {"error":"Operator","message":"Operator: LoadingInfoError: UnknownDatasetId"}

how do you run your backend?

[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply examples/add_public_raster_dataset.ipynb:355]: Oh, I just meant the example notebook without running it does not contain the output.

[the remark this answers is not in the document]
Comment (michaelmattig) [reply examples/add_public_raster_dataset.ipynb:363]: 24e878fccd1e0456464c9b7c73f4e83c10f064bd

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request closed without merging**